packages/airnode-node/src/api/index.ts

```diff
@@ -135,7 +134,7 @@ function verifyRequestId(payload: CallApiPayload): LogsData<ApiCallErrorResponse
 
 export function verifyTemplateId(payload: CallApiPayload): LogsData<ApiCallErrorResponse> | null {
```

Comment:
So, this check will basically never fail for the HTTP signed data request, right? But I'm ok with doing the check anyway, I just want to make sure I understand.

Reply:
yes, with the way it is currently it will always pass. If we decide to pass in templateId at some point then the check will be useful.